test/resources/tracker.js

```diff
@@ -31,4 +31,19 @@ describe('Tracker Resource', () => {
       expect(err).to.be.an.instanceOf(NotImplementedError);
     });
   });
+
+  describe('Trackers in bulk', () => {
```

Comment:
Did you actually run this unit test? Our Node library is a bit different in the fact that the unit test suite makes 0 API calls and everything is mocked (we'll be fixing this shortly). I don't know that we had a way built out for the tests to accept an API key to actually make this call here.

Reply:
Yes, I just E2E tested it.